VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/models/participant-panel.model.spec.ts

```diff
@@ -38,6 +39,27 @@ describe('ParticipantPanelModel', () => {
         expect(model.isAvailable()).toBeTruthy();
     });
 
+    it('returns isAvailable: true when participant is quick link user and status is in consultation', () => {
+        participant.status = ParticipantStatus.InConsultation;
+        model = mapper.mapFromParticipantUserResponse(participant);
+        spyOnProperty(model, 'isQuickLinkUser').and.returnValue(true);
+        expect(model.isAvailable()).toBeTruthy();
```

Comment:
Use toBe(true) or toBeTrue()(and toBe(false) or toBeFalse()) as opposed to Truthy and Falsy when testing a boolean